testing/p0f/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
+# Maintainer: Arch3y <arch3y@riseup.net>
+pkgname=p0f
+pkgver=3.09b
+pkgrel=0
+pkgdesc="Passive traffic fingerprinting tool"
+url="http://lcamtuf.coredump.cx/p0f3/"
+arch="all"
+license="GPL2"
+makedepends="bash libpcap-dev"
+subpackages="$pkgname-doc"
+source="http://lcamtuf.coredump.cx/p0f3/releases/$pkgname-$pkgver.tgz p0f-fixes.patch"
+
+build() {
+	cd "$srcdir/$pkgname-$pkgver"
+	make
+	cd tools
+	make p0f-client
+	make p0f-sendsyn
+	make p0f-sendsyn6
+}
+
+package() {
+	cd "$srcdir/$pkgname-$pkgver"
+        install -Dm 755 p0f "$pkgdir/usr/bin/p0f"
```

Comment:
This seems to have some inconsistent indentation. Also, you can use `cp` if the existing permissions are correct.